sails/api/controllers/users/get-subscriptions.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+
+  friendlyName: 'Get IDs of boards user have subscribed',
+
+  description: 'Get IDs of boards user have currently subscribed',
+
+  inputs: { },
+
+  exits: {
+    success: {
+      responseType: '',
+      statusCode: 200
+    },
+    missingParams: {
+      description: 'Missing the ID of the user',
+      statusCode: 400
+    }
+  },
+
+  fn: async function(inputs, exits) {
+    if(!this.req.me['id']) {
+      return exits.missingParams();
```

Comment:
Should return a 401 unauthorized instead. There is no missing parameter that could be sent, but authorization is missing